### ISSUE TITLE
Blocking behaviour of uploader

### ISSUE THREAD

**Issue by RadRussianRus** [telegramdesktop/tdesktop#5394]
When you select a file to upload, Telegram shows popup window, where you can add caption (or add other files). This popup blocks all other functions of Telegram: you can't switch to other chat, you can't even copy something from current chat, and then return to caption. So some kind of non-blocking behavior for this popup could be useful. Especially since limit for captions was increased to 1024.

**Comment by Remu-rin** [comment]
Yes, something like behavior of the exporting data popup would be great.

**Comment by Lonami** [comment]
> can't switch to other chat

Fun fact you actually can press <kbd>ESC</kbd> or <kbd>Ctrl+TAB</kbd> or some other combination I don't remember now to select the background :p

**Comment by RadRussianRus** [comment]
@Lonami 
Yeah, but it seems like very tricky way to do it.

**Comment by john-preston** [comment]
Currently all in-app boxes and layers (like settings / profiles) are blocking.

**Comment by RadRussianRus** [comment]
@john-preston
Data export popup is non-blocking. Will some kind of this behavior be implemented for file/image uploader at any point of time?

**Comment by Remu-rin** [comment]
Not resolved.

**Comment by Lonami** [comment]
Bump (seems the bot only listens to contributors?).

**Comment by RadRussianRus** [comment]
No need to close it.

**Comment by EricKotato** [comment]
Update on issue. This issue partially resolved, since you can open chats in separate windows, but since you can neither open multiple windows for the same chat, nor you can temporarily hide the uploader popup, the issue still stands.
